### PR TITLE
Allow building image for any ARCH

### PR DESCRIPTION
**Pull request by synarete**
Do not assume specific hard-coded ARCH when building Docker image.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,21 +14,21 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 COMMIT_ID = $(shell git describe --abbrev=40 --always --dirty=+ 2>/dev/null)
 GIT_VERSION = $(shell git describe --match='v[0-9]*.[0-9].[0-9]' 2>/dev/null || echo "(unset)")
 
-CONFIG_KUST_DIR:=config/default
-CRD_KUST_DIR:=config/crd
-MGR_KUST_DIR:=config/manager
-KUSTOMIZE_DEFAULT_BASE:=../default
+CONFIG_KUST_DIR := config/default
```
Comment (phlogistonjohn) [inline Makefile:17]: I approve of consistency, but I'd prefer it to be consistently without spaces. Would you object to that?

File: Dockerfile [hunk 2]
```diff
@@ -2,6 +2,7 @@
 FROM docker.io/golang:1.17 as builder
 ARG GIT_VERSION="(unset)"
 ARG COMMIT_ID="(unset)"
+ARG ARCH="(unset)"
```
Comment (phlogistonjohn) [inline Dockerfile:5]: The string "(unset)" has no special meaning and I used it to pass strings to the go binary so we'd know if the build never specified the GIT version and id. However I'm not sure it makes sense to set ARCH to "(unset)"  would it make more sense to default to empty string "" or "amd64" ?  Just so it defaults to something valid?

Comment (synarete) [reply Dockerfile:5]: I was wondering about it as well. I would vote for empty string. Will fix.

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# Alllow developer to override some defaults
+-include devel.mk
```
Comment (phlogistonjohn) [inline Makefile:2]: Great! I learned something today. (Dash prefix to ignore errors when the files doesn't exist)

**Review by phlogistonjohn — APPROVED** [review]

**Pull request merged**